### ISSUE TITLE
OpenGraph doesn't correctly detect encoding or fails if site is encoded wrongly

### ISSUE THREAD

**Issue by jhass** [diaspora/diaspora#5205]
Not sure which one is the case. In the former case we should patch upstream, in the later silently ignore the error.

```
Workers::GatherOpenGraphData

288941, "http://jetzt.sueddeutsche.de/texte/anzeigen/588670/Das-ist-vom-Grundgesetz-nicht-gedeckt"

app/models/open_graph_cache.rb:25:in `new'
app/models/open_graph_cache.rb:25:in `fetch_and_save_opengraph_data!'
app/models/open_graph_cache.rb:20:in `find_or_create_by'
app/workers/gather_open_graph_data.rb:12:in `perform'
```

**Comment by jhass** [comment]
Should've been fixed by #5347 and cleaned up by ed0005c44d12e51629481f67164d6fee470841c3

**Comment by jhass** [comment]
Seems to be fixed, probably due to #5462
